rest-api-spec/src/main/resources/rest-api-spec/test/ingest/80_dedot_processor.yaml

```diff
@@ -0,0 +1,33 @@
+---
+"Test De-Dot Processor":
+  - do:
+      ingest.put_pipeline:
+        id: "my_pipeline"
+        body:  >
+          {
+            "description": "_description",
+            "processors": [
+              {
+                "dedot" : {
+                  "separator" : "_"
```

Comment:
if you pass in a separator, maybe pass in one that's not default? otherwise this test doesn't prove that the separator param actually works. I would leave this test without separator and add another one with a non default one.